readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://sich-mykhailo.github.io/layout_search-bar-airbnb/)
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/214536201-48f42a8c-f42d-4f2b-af4d-218dd0b0a4a8.png)
There is no such dark outline in the design

Reply:
done